File: java/src/main/be/steinsomers/bron_kerbosch/BronKerbosch2_gp.java
package be.steinsomers.bron_kerbosch;

public final class BronKerbosch2_gp implements BronKerboschAlgorithm {
    @Override
    public void explore(final UndirectedGraph graph, final CliqueConsumer cliqueConsumer) {
        BronKerboschPivot.explore(graph, cliqueConsumer, PivotChoice.MaxDegreeLocal);
    }
}

File: kotlin/src/main/be/steinsomers/bron_kerbosch/BronKerbosch2gp.kt
package be.steinsomers.bron_kerbosch

class BronKerbosch2gp : BronKerboschAlgorithm {
    override fun explore(graph: UndirectedGraph, cliqueConsumer: (IntArray) -> Unit) {
        BronKerboschPivot.explore(graph, cliqueConsumer, PivotChoice.MaxDegreeLocal)
    }
}
